enterprise/backend/src/metabase_enterprise/advanced_config/file.clj

```diff
@@ -1,5 +1,4 @@
-(ns ^{:added "0.45.0"}
- metabase-enterprise.config-from-file.core
+(ns ^{:added "0.45.0"} metabase-enterprise.advanced-config.file
```

Comment:
👍 on versioning the ns. 
Should we start doing this for source files? or just ee or feature-focused ns?

Reply:
It never really hurts to have this information, so you can see at a glance when a new feature was added. We try to do it for driver methods and stuff like that but there's no reason not to do it for other features too. If you add a new namespace with a new feature then you can add this info if you want